get-started/dedicated-endpoints.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Launch dedicated endpoints
+description: Learn how to spin up and call private model deployments on kluster.ai with predictable per hour pricing.
+---
+
+# Dedicated deployments
+
+Dedicated deployments let you run a private instance of any [Hugging Face text model](https://huggingface.co/models){target=\_blank} on hardware reserved just for you. Enjoy full control, predictable per‑minute billing, and zero per‑token costs.
+
+This page covers how to create, use, and stop your dedicated deployments.
+## How to create a deployment
+
+Ensure you're logged in to the [kluster.ai platform](https://platform.kluster.ai){target=\_blank}, then navigate to [Dedicated deployments](https://platform.kluster.ai/dedicated-deployments){target=\_blank} page, then press **Launch Deployment**.
+
+![Launch deployment](/images/get-started/dedicated-endpoints/dedicated-1.webp)
+
+Then, complete the following fields to configure your deployment:
+
+1. **Deployment Name**: Enter a clear Deployment Name (e.g., `mydedicated`) so you can spot it later in the console.
+2. **Model selection**: Paste the Hugging Face model ID or URL (e.g., `deepseek-ai/DeepSeek-R1`). If the model is private, provide a Hugging Face access token.
+3. **Select Hardware**: Confirm a GPU configuration.
+4. **Specify auto-shutdown**: Set an auto‑shutdown window for your instance to power down after a specified period of inactivity, between 15 minutes to 12 hours. 
+4. **Launch**: Review the estimated price and then Click **Launch deployment**. Spin‑up takes ≈10–20 min; once the status shows `Running`, copy the Endpoint ID, as you'll use that to submit requests. 
```

Comment:
```suggestion
4. **Launch**: Review the estimated price and then click **Launch deployment**. Spin‑up takes ≈10–20 min; once the status shows `Running`, copy the Endpoint ID, as you'll use that to submit requests. 
```

Is Endpoint ID a UI element? If so, please put it in bold, otherwise no need to capitalize Endpoint